wled00/pin_manager.cpp

```diff
@@ -32,6 +32,11 @@ bool PinManagerClass::deallocatePin(byte gpio, PinOwner tag)
     return false;
   }
 
+  #ifdef SOC_TOUCH_VERSION_2 // ESP32 S2 and S3 have a fucntion to check touch state, detach any previous assignments
```

Comment:
I do not think I like this approach here. PinManager never attached any interrupts/interrupt handlers to GPIO so it shouldn't detach any as well.
If you need to detach an interrupt handler it should be done at the place where such handler is attached (`set.cpp` only as there is no counterpart for boot sequence).